README.md

```diff
@@ -283,7 +301,7 @@ PerfKitBenchmaker.
 Flag | Notes
 -----|------
 `--help`       | see all flags
-`--cloud`      | Check where the bechmarks are run.  Choices are `GCP`, `AWS`, `Azure`, or `DigitalOcean`
+`--cloud`      | Check where the bechmarks are run.  Choices are `GCP`, `AWS`, `Azure`, `DigitalOcean`, or `OpenStack`
```

Comment:
Add the default zone to the table below?

Alternatively, replace it with this to avoid duplication:

`--cloud`      | Cloud provider where the benchmarks are run, for example `--cloud=GCP`. See the table below for choices.
`--zone`       | This flag allows you to override the default zone. See below for cloud-specific defaults.